website/content/ja/docs/installation.md

```
@@ -28,15 +31,15 @@ $ wget https://github.com/manaelproxy/manael/releases/download/v1.x.y/manael_1.x
 $ tar xf manael_1.x.y_Linux_x86_64.tar.gz
 ```
 
-### 3. インストール {#3-install}
+### 3. インストール {#install}
 
 ファイルをコピーするために `install` コマンドを利用します。`cp` コマンドや `mv` コマンドでも同様の作業はできますが、`install` コマンドを使うことによって適切な実行権限が実行ファイルに与えられます。
 
 ```console
 $ sudo install manael /usr/local/bin
```

markdownlint-cli2 (0.21.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)